packages/gatsby/src/utils/eslint-rules/limited-exports-page-templates.ts

```diff
@@ -0,0 +1,80 @@
+import { Rule } from "eslint"
```

Comment:
I assume we don't care about calling `grapqhl` as a regular call? (`graphql('my query')` works all the same)

Reply:
afaik we only support the TaggedTemplateExpression in page templates, the `graphql()` only works in gatsby-node.js